hepcrawl/parsers/elsevier.py

```diff
@@ -341,11 +342,17 @@ def page_end(self):
 
     @property
     def publication_date(self):
+        publication_date = None
         publication_date_string = self.root.xpath(
             './RDF/Description/coverDisplayDate/text()'
         ).extract_first()
         if publication_date_string:
-            publication_date = PartialDate.parse(publication_date_string)
+            try:
+                publication_date = PartialDate.parse(publication_date_string)
+            except:
+                # in case when date contains month range, eg. July-September 2020
+                publication_date = re.sub("[A-aZ-z]*-(?=[A-aZ-z])", "", publication_date_string)
```

Comment:
so `July-September 2020` becomes 2020? that's not really great.

Reply:
No, it leaves the second month from the range.